### DISCUSSION TITLE
Wyze Rules for Home or Away

### DISCUSSION THREAD

**Discussion by DdraigX** [mrlt8/docker-wyze-bridge#1274] [Ideas]
Forgive me if this is already a function I don't see it available anywhere. I'm using wyze bridge for Home Assistant. I have two rules configured for being away or being home. Does the wyze api allow you to execute rules? It would be a great feature to execute specific rules or even have a button that would turn on or off the rules with in the wyze app.

**Comment by mrlt8** [comment]
I believe there are a couple of other projects that may already have rules integrated.

I personally haven't used any rules, but would a single rule be tied to a single camera or would they apply to multiple cameras?

**Comment by DdraigX** [comment]
The rules are usually associated with multiple cameras. I've attached a screenshot of my I'm away rule. For my use case something that would just act as a "button press" to turn on the I'm Away Rule, or the I'm Home rules that I already have established in Wyze would be sufficient. 

![Screenshot_20240625_115239_Wyze](https://github.com/mrlt8/docker-wyze-bridge/assets/6390496/285a4f9a-53d3-4314-9dd6-6b11034a4dd6)

**Reply by mrlt8** [reply]
Thanks! Rules seem a little outside the scope of this project, but I can try to add motion and notification toggles.

**Comment by DdraigX** [comment]
Thanks!

In HA I was able to create some motion detection alerts so don't spend too
much time on it. I was just curious if the wyze api would allow it to
execute one of their pre-define rules and if that could be am option. So
when I'm away from my homezone HA could automatically execute that rule.

**Reply by mrlt8** [reply]
I've add a `notifications` and `motion_detection` API/MQTT endpoint that should be able to toggle those two options in the latest edge build. 

FYI, `notifications` is currently cloud based, so you'll need to do a GET on the topic to update the status and it will only work on cameras registered under that account and will NOT work with shared cameras. I'm still looking to see if there's a way to control the notifications locally though.
